.buildkite/scripts/create-github-release.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+# Download the signed artifacts and run the GitHub release using the GH cli
+#
+# Required environment variables:
+#  - BUILDKITE_TAG
+#
+set -eox pipefail
+
+DIST_LOCATION=artifacts-to-upload
+
+echo "--- Download signed artifacts"
+mkdir -p "$DIST_LOCATION"
+buildkite-agent artifact download --step gpg "*.*" "$DIST_LOCATION"/
+buildkite-agent artifact download --step macos "*.*" "$DIST_LOCATION"/
+buildkite-agent artifact download --step windows "*.*" "$DIST_LOCATION"/
+ls -l "$DIST_LOCATION"/
+
+echo "--- Install gh :github:"
+if ! gh --version &>/dev/null ; then
+  wget -q https://github.com/cli/cli/releases/download/v2.50.0/gh_2.50.0_linux_amd64.tar.gz -O gh.tar.gz
+  tar -xpf gh.tar.gz --strip-components=2
+  PATH="$(pwd):${PATH}"
+  export PATH
+  gh --version
+fi
+
+echo "--- Run GitHub release"
+if [ -n "${BUILDKITE_TAG}" ] ; then
+  gh release \
+    create \
+    "${BUILDKITE_TAG}" \
+    --draft \
+    --title "${BUILDKITE_TAG}" \
+    --repo elastic/synthetics-recorder \
+    "${DIST_LOCATION}/*.*"
+else
+  echo "gh release won't be triggered this is not a Git tag release"
+fi
```

Comment:
nit: wondering if it's just easier to create a release directly with

```
curl -L \
  -X POST \
  -H "Accept: application/vnd.github+json" \
  -H "Authorization: Bearer <YOUR-TOKEN>" \
  -H "X-GitHub-Api-Version: 2022-11-28" \
  https://api.github.com/repos/OWNER/REPO/releases \
  -d '{"tag_name":"v1.0.0","target_commitish":"master","name":"v1.0.0","body":"Description of the release","draft":false,"prerelease":false,"generate_release_notes":false}'
  ```

Reply:
I copied the script from https://github.com/elastic/synthetics-recorder/blob/main/.ci/scripts/create-github-release.sh.

In addition, it does upload the artifacts too, see `"${DIST_LOCATION}/*.*"`

Reply:
Ah, right. lgtm